src/views/home/sections/Landing.module.scss

```diff
@@ -25,6 +25,8 @@
 		min-height: 55vh;
 		max-height: 70vh;
 
+		padding: 2rem;
+
```

Comment:
Because of the increased length of the title, we actually need to increase the width of the container. The current view causes the title to wrap on narrower views.

Reply:
Decreased `h1` font size instead in cd35876.